src/lumigo_tracer/auto_instrument_handler.py

```diff
@@ -10,13 +10,13 @@ def parse_handler():
     try:
         module_name, unit_name = os.environ[ORIGINAL_HANDLER_KEY].rsplit(".", 1)
     except KeyError:
-        raise ValueError(
+        raise Exception(
             "Could not find the original handler. Please contact Lumigo for more information."
-        )
-    except ValueError:
-        raise RuntimeError(
-            f"Invalid handler format: Bad handler '{os.environ[ORIGINAL_HANDLER_KEY]}': not enough values to unpack (expected 2, got 1)"
-        )
+        ) from None
```

Comment:
Why are you adding `from None`? The user will lose the context of the exception:
![image](https://user-images.githubusercontent.com/49228804/103617315-bb0dbc80-4f36-11eb-8924-4695a018eba9.png)

Same regarding the other `from None` is this file

Reply:
Yeah, That's what AWS does, and I want to have the same behavior...

Reply:
Yep, I tried to copied AWS behavior:
![image](https://user-images.githubusercontent.com/47108628/103623964-89e6b980-4f41-11eb-8c46-d2dc0fda6b6a.png)